src/Raven.Studio/typescript/components/common/DatabaseGroup.tsx

```diff
@@ -76,11 +78,11 @@ export function DatabaseGroupType(props: DatabaseGroupTypeProps) {
 
         switch (type) {
```

Comment:
please check what type this method returns? string or IconName?

Reply:
Now it returns `IconName`